includes/opentsdb.inc.php

```diff
@@ -18,9 +18,12 @@ function opentsdb_update($device, $measurement, $tags, $fields)
     if ($config['opentsdb']['enable'] == true) {
         if ($opentsdb != true) {
             $opentsdb = fsockopen($config['opentsdb']['host'], $config['opentsdb']['port']);
-            d_echo("Connection made to OpenTSDB");
+            d_echo("Connection made to OpenTSDB\n");
+        }
+        if ($opentsdb == true) {
```

Comment:
Probably easier to do `} else {` from the `if ($opentsdb != true) {` and do the echo.